.github/workflows/ci.yml

```diff
@@ -712,9 +714,19 @@ jobs:
               ignore exclusion if: >-
                 ${{ needs.build-pt.outputs.have_gcp_secrets == 'true' }}
 
-            - suite: suite-delta-lake-databricks
+            - suite: suite-delta-lake-databricks73
               config: hdp3
-            - suite: suite-delta-lake-databricks
+            - suite: suite-delta-lake-databricks73
+              ignore exclusion if: >-
+                ${{ needs.build-pt.outputs.have_databricks_secrets == 'true' }}
+            - suite: suite-delta-lake-databricks91
+              config: hdp3
+            - suite: suite-delta-lake-databricks91
+              ignore exclusion if: >-
+                ${{ needs.build-pt.outputs.have_databricks_secrets == 'true' }}
```

Comment:
Do I read correctly that we have two Configs for databricks test and the `default` config requires secrets while the `hdp3` config doesn't require secrets?

Reply:
See https://github.com/trinodb/trino/actions/runs/3220361174/jobs/5266972748 , step `Build PT matrix (all)`.
I think that `hdp3` is actually irelevant. I changed the `config` to `default` instead of `hdp3`.

@nineinchnick  CPTAL whether the `config` `hdp3` still makes any sense?

Reply:
I don't know, do delta lake tests rely on the hdp3 platform in any way?

Reply:
I think the delta lake tests should've been added to a separate matrix (or exclude every config except default) since the beginning. We can do that now probably.

Reply:
I need some extra pointers here to go forward.

Reply:
I meant that the matrix includes a `config` as well at https://github.com/trinodb/trino/pull/14549/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR681-R683.

It has two values.

The `suite` entry has one for each databricks version. So overall result of matrix is 2 jobs per databricks version - one with ConfigDefault and other with ConfigHdp3.

Now AFAIK DeltaLake doesn't depend on version of HMS/HDFS being used so it doesn't matter if we run it using ConfigDefault or ConfigHdp3. So IMO those two configs are redundant for Delta Lake.

So a possible solution is to exclude the hdp3 config for DeltaLake suites.
There might be multiple ways to do that - the way I could think of is to create a separate "job" with:

```yml
matrix:
  config:
    - default
  suite:
    - suite-delta-lake-databricks73
    - suite-delta-lake-databricks91
    - ....
```